backend/controllers/indUsersController.js

```diff
@@ -0,0 +1,25 @@
+const IndUser = require('../models/indUserModel');
+const mongoose = require('mongoose');
+
+//get all individual users
+const getAllIndUsers = async (req, res) => {
+    const indUsers = await IndUser.find({}).sort({createdAt: -1})
+    res.status(200).json(indUsers)
+}
+
+//create a new individual user
+const createIndUser = async (req,  res) => {
+    const {firstName, surname, email, password} = req.body
```

Comment:
Here if we use firstName then 'surname' should be 'last name' so: 
`{firstName, lastName, email, password}`
or
`{forename, surname, email, password}`

It's a english language rule that I was told but I don't remember the exact reason why

Reply:
I ended up merging backend to frontend locally yesterday so we could keep working, so I'll change this in HO-38 and amend that PR as as it's not been approved yet

Reply:
Have gone with lastName, FYI